#Header header    # ROS Message headaer

int8 target     # target

int32 command   # Command ID
uint8 axis      # Axis number
float32 fval    # Float value 
float32 fval2   # Float second value (in case of dual axis command)

